diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/constants/Warnings.kt

```diff
@@ -70,6 +70,7 @@ enum class Warnings(private val id: Int, private val canBeAutoCorrected: Boolean
     BLANK_LINE_BETWEEN_PROPERTIES(50, true, "there should be no blank lines between properties without comments; comment or KDoc on property should have blank line before"),
     BRACES_BLOCK_STRUCTURE_ERROR(51, true, "braces should follow 1TBS style"),
     WRONG_INDENTATION(52, true, "only spaces are allowed for indentation and each indentation should equal to 4 spaces (tabs are not allowed)"),
+    EMPTY_BLOCK_STRUCTURE_ERROR(53, true, "empty block shouldn't be use, until function with annotation override"),
```

Comment:
```suggestion
    EMPTY_BLOCK_STRUCTURE_ERROR(53, true, "incorrect format of empty block"),
```
We don't really warn if empty block is used in general, we warn if newline is used incorrectly inside of empty block or it is used in improper combination with other structures.
Please also add it to available-rules.md.